Bucketeer/Sources/Internal/Remote/RequestBody/RegisterEventsRequestBody.swift

```diff
@@ -1,6 +1,16 @@
 import Foundation
 
 struct RegisterEventsRequestBody: Codable {
+    internal init(events: [Event] = [],
+                  sdkVersion: String = Version.current,
+                  sourceId: SourceID = SourceID.ios
```

Comment:
set the default values. it like the way we did in Android SDK 
https://github.com/bucketeer-io/android-client-sdk/pull/165/files#diff-4d7bb55f81bf07536550c06fc990b013fd606b6054932e3c88bb9c60ee44cbd4R12